resources_pages/install_ds_stack_windows.md

```diff
@@ -3,290 +3,540 @@ layout: page
 
-Unpack the archive and copy it into the %userprofile%\AppData\Roaming\GNU MCU Eclipse (for example C:\Users\\`user_name`\AppData\Roaming\GNU MCU Eclipse) folder. 
+Next we need to add the `bin` folder in this extraction location to our PATH so that we can use `make` from the terminal (like we did with R earlier). Open the bash configuration file with VS Code by pasting this into a terminal:
 
-Next, like you did when installing [Rscript](https://github.com/UBC-MDS/UBC-MDS.github.io/blob/master/resources_pages/install_ds_stack_windows.md#rscript), you need to add the folder where `make.exe` is located to the _PATH_ variable. The `make.exe` file is located in C:\Users\\`user_name`\AppData\Roaming\GNU MCU Eclipse\Build Tools\2.12-20190422-1053\bin.
+```
+code ~/.bash_profile
+```
+
+And replace the section that reads:
+
+```
+# Add R and Rscript to path
+export PATH="/c/Program Files/R/R-4.0.2/bin/x64":$PATH
+```
 
-After installation, restart Powershell and type the following to ask for the version:
+with the following (prepending the `make` folder to the path above):
+
+```
+# Add R, Rscript, and Make to path
+export PATH="/c/Users/${USERNAME}/Documents/make-4.3-without-guile-w32-bin/bin":"/c/Program Files/R/R-4.0.2/bin/x64":$PATH
+```
+
+Launch a new terminal and run
 
 ```
 make --version
 ```
 
-you should see something like this if you were successful:
+which should return something like
 
 ```
-GNU Make 4.2.1
-Built for i686-pc-msys
-Copyright (C) 1988-2016 Free Software Foundation, Inc.
+GNU Make 4.3
+Built for Windows32
+Copyright (C) 1988-2020 Free Software Foundation, Inc.
 License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
 This is free software: you are free to change and redistribute it.
 There is NO WARRANTY, to the extent permitted by law.
 ```
 
+## PostgreSQL
+
+We will be using PostgreSQL as our database management system. You can [download PostgreSQL from here](https://www.enterprisedb.com/downloads/postgres-postgresql-downloads). Follow the instructions for the installation. In the password page, type whatever password you want, but make sure you'll remember it later. For all the other options, use the default. You do not need to run "StackBuilder" at the end of the installation.
+
+To test if the installation was successful open the `SQL Shell` app from the Start menu. You will be asked to setup your configuration, accept the default value (the one within square brackets) for the first four values by pressing enter four times, then type in your password and press enter one last time. It should look like this if it is working correctly:
```

Comment:
Make them run through the config so they understand what it is and practice remembering their passworkd